core/src/test/scala/com/nvidia/spark/rapids/tool/util/ToolUtilsSuite.scala

```diff
@@ -199,6 +200,20 @@ class ToolUtilsSuite extends FunSuite with Logging {
     result.value.get shouldBe (59 * 1000 + 200)
   }
 
+  /**
+   * Reads the content of a file as UTF-8 and closes the resources.
+   * @param filePath Path to the file
+   * @return Content of the file as a string
+   */
+  private def readFileContentAsUtf8(filePath: String): String = {
+    val source = Source.fromFile(filePath)(StandardCharsets.UTF_8)
+    try {
+      source.getLines().mkString("\n")
+    } finally {
+      source.close()
+    }
+  }
+
```

Comment:
Thanks @parthosa 
should we move this to one of the util objects. Ideally, we should replace all `Source.fromFile(` with this new method and put a scalastyle rule to verify that it won't be added back.
I am fine if this PR is not fixing the other `Source.fromFile`, but having it in a util objects will be good enough to fix the other calls later.

Reply:
Thanks @amahussein. I agree. Let me update the changes.

Reply:
Added a wrapper object `UTF8Source` and an entry in `scalastyle_config.xml`